tests/unit/test-class-wc-payments-token-service.php

```diff
@@ -175,7 +175,6 @@ public function test_add_token_to_user_for_sepa() {
 	 * Test add SEPA token to user with deferred intent creation UPE.
 	 */
 	public function test_add_token_to_user_for_sepa_deferred_intent_creation_upe() {
-		$this->mock_cache->method( 'get' )->willReturn( [ 'is_deferred_intent_creation_upe_enabled' => true ] );
```

Comment:
This flag was not checked anywhere in the codebase - likely a leftover from a cleanup task.